api/source/job_control/create_job.rst

```diff
@@ -78,6 +78,12 @@ Creating a job is prerequisite for virtually all other methods.
 |                   +------------------+-----------------------------------------------------------+
 |                   | `Example`        | ``expected_speakers=40``                                  |
 +-------------------+------------------+-----------------------------------------------------------+
+| is_duplicate      | `Description`    | Bypass the ducplicate detection settings                  |
```

Comment:
```suggestion
| is_duplicate      | `Description`    | Bypass the duplicate detection settings                  |
```

Reply:
Here you can use the same phrase that the suggested one: `Allow creating a job with the same settings that another job (duplicated job`

Reply:
Addressed